compiler/rustc_hir_analysis/src/check/intrinsicck.rs

```diff
@@ -44,6 +44,28 @@ impl<'a, 'tcx> InlineAsmCtxt<'a, 'tcx> {
         false
     }
 
+    fn simd_to_asm_type(&self, ty: Ty<'tcx>, size: u64) -> Option<InlineAsmType> {
+        match ty.kind() {
+            ty::Never | ty::Error(_) => return None,
```

Comment:
Early returning at this level of control flow, being inside a different function, isn't the same as before. Just remove this and let the wildcard happen.

Reply:
```suggestion
```